### DISCUSSION TITLE
Didn't get the benifits after the request is approved on Github student Devoloper Pack

### DISCUSSION THREAD

**Discussion by debraj17** [community/community#128617] [GitHub Education]
### Hi everyone,

I apply for github student developer pack on 12 june and the application got approved by github but didn't get the benefits why is this showing tell me please.
![Screenshot 2024-06-15 at 11-50-23 GitHub Education](https://github.com/community/community/assets/163664655/762baf3d-92a2-458e-a0c2-f3b74016a08e)

**Comment by davevad93** [comment]
Hi @debraj17 ,

Your request was approved but your benefits will be available soon, which means that you still have to wait a few days for your benefits to be fully available.

![0](https://github.com/community/community/assets/94308234/0830e2ef-6ca9-4e75-bc0c-a6cd56bc5354)

**Reply by debraj17** [reply]
Thanks sir
